src/utilities/filetypes/Filetypes.i

```diff
@@ -18,9 +18,12 @@
   %ignore openstudio::WorkflowStepValue::WorkflowStepValue(std::string const&, char const*);
   %ignore openstudio::MeasureStep::setArgument(std::string const&, char const*);
 
-  // Ignore Json::Value return type
+  // Ignore Json::Value return type (toJSON / fromJSON are globally ignored already)
   %ignore openstudio::StandardsJSON::getPrimaryKey;
 
+  %ignore openstudio::CustomOutputAdapter::CustomOutputAdapter;
+  %ignore openstudio::CustomOutputAdapter::optionsJSON;
```

Comment:
Ignore other Json::Value related things in Filetypes.i when C#

Reply:
This will unbrick the develop build, which fails due to this problem currently, after the merge of #4917 (which we didn't realize it was going to break the build because the CI incremental windows runner does not build c# bindings...), cf https://ci.openstudio.net/blue/organizations/jenkins/openstudio-develop-full/detail/openstudio-develop-full/360/pipeline/98#step-101-log-921

```
D:\OSN\build\csharp_wrapper\generated_sources\OpenStudioUtilitiesData\SWIGTYPE_p_Json__Value.cs(13,14): 
error CS0101: The namespace 'OpenStudio' already contains a definition for 'SWIGTYPE_p_Json__Value' [D:\OSN\build\csharp_wrapper\OpenStudio.csproj]

D:\OSN\build\csharp_wrapper\generated_sources\OpenStudioUtilitiesFileTypes\SWIGTYPE_p_Json__Value.cs(13,14): 
error CS0101: The namespace 'OpenStudio' already contains a definition for 'SWIGTYPE_p_Json__Value' [D:\OSN\build\csharp_wrapper\OpenStudio.csproj]

D:\OSN\build\csharp_wrapper\generated_sources\OpenStudioUtilitiesData\SWIGTYPE_p_Json__Value.cs(16,12): error CS0111: Type 'SWIGTYPE_p_Json__Value' already defines a member called 'SWIGTYPE_p_Json__Value' with the same parameter types [D:\OSN\build\csharp_wrapper\OpenStudio.csproj]

D:\OSN\build\csharp_wrapper\generated_sources\OpenStudioUtilitiesData\SWIGTYPE_p_Json__Value.cs(20,13): error CS0111: Type 'SWIGTYPE_p_Json__Value' already defines a member called 'SWIGTYPE_p_Json__Value' with the same parameter types [D:\OSN\build\csharp_wrapper\OpenStudio.csproj]

D:\OSN\build\csharp_wrapper\generated_sources\OpenStudioUtilitiesFileTypes\SWIGTYPE_p_Json__Value.cs(16,12): error CS0111: Type 'SWIGTYPE_p_Json__Value' already defines a member called 'SWIGTYPE_p_Json__Value' with the same parameter types [D:\OSN\build\csharp_wrapper\OpenStudio.csproj]

```